<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulatorControlWindow</class>
 <widget class="QDialog" name="SimulatorControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator dialog</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="SimulatorTimes">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Simulator times</string>
      </property>
      <widget class="QLineEdit" name="SimulatedTime">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="SimulatedTimeLabel">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Simulated (ms)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>User</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>System</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="timeSystem">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="timeUser">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Control">
      <property name="title">
       <string>Simulator handling</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="resetButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Simulatingmode">
      <property name="title">
       <string>Simulating mode</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QRadioButton" name="continuousMode">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Continuous</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="spikeMode">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Spike</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="eventMode">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Event</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="stepMode">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Step</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="timeMode">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="displayModeBox">
      <property name="title">
       <string>Display mode</string>
      </property>
      <widget class="QRadioButton" name="displayContinuous">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Continuous</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Spike</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Waiting...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="method1Button">
        <property name="text">
         <string>Method1</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="method2Button">
        <property name="text">
         <string>Method2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="method3Button">
        <property name="text">
         <string>Method3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
